agent-install/agent-uninstall.sh

```diff
@@ -242,14 +242,11 @@ function removeNodeFromLocalAndManagementHub() {
         fi
     else
         log_info "Node is not registered, skip unregister..."
-        if [[ "$DELETE_EX_NODE" == "true" ]]; then
-            log_info "Remove node from the management hub..."
-            deleteNodeFromManagementHub $NODE_ID
-        fi
     fi
 
```

Comment:
You have moved the `deleteNodeFromManagementHub` out of the `else` statement. Was it intentional?

Reply:
I see that `unregister()` function does try to remove node it the flag `DELETE_EX_NODE` is true. So I am not sure if this change is going to work properly in all scenarios.